android/src/main/java/com/reactnativespokestack/SpokestackModule.kt

```diff
@@ -36,6 +37,15 @@ class SpokestackModule(private val reactContext: ReactApplicationContext): React
     CLASSIFY
   }
 
+  private enum class Filenames(val value: String) {
```

Comment:
There'd be slightly less overhead if these were just `const`s, but it's not a big deal.

Reply:
I was playing with looping them to remove them, but I realized later all I needed to do was overwrite them.